vscode-trace-extension/src/extension.ts

```diff
@@ -102,3 +110,21 @@ export function activate(context: vscode.ExtensionContext): void {
 export function deactivate(): void {
     traceLogger.disposeChannel();
 }
+
+async function startTraceServerIfAvailable(): Promise<void> {
+    const extensionId = 'vscode-trace-server';
+    const traceServerExtension = vscode.extensions.getExtension('tracecompass-community.' + extensionId);
+    if (!traceServerExtension || await isUp()) {
+        return;
+    }
+    await vscode.commands.executeCommand(extensionId + '.start-if-stopped');
+}
+
+async function isUp() {
+    if (tspClientProvider) {
```

Comment:
Shouldn't need this check now, as the `tspClientProvider` is initialized unconditionally and never reassigned.

Reply:
I can change that too.